### PR TITLE
Update Veracode Example.postman_collection.json

### PR DESCRIPTION
**Pull request by mhowes-veracode**
Added calls for GET all static findings and GET all SCA findings. In the [REST API Quickstart](https://docs.veracode.com/r/REST_APIs_Quickstart), under [Retrieve Scan Results](https://docs.veracode.com/r/REST_APIs_Quickstart#optional-retrieve-scan-results), we tell users to create these calls. With these included, they only need to add their `app_guid` and select **Send**.

For example:

image

### CODE REVIEW

**Comment by mhowes-veracode** [comment]
There's an older PR for this file that should be merged before this one: https://github.com/veracode/veracode-postman/pull/8

File: Veracode Example.postman_collection.json
Comment (tjarrettveracode) [inline Veracode Example.postman_collection.json:3]: very interesting, wonder what information we're leaking there

File: Veracode Example.postman_collection.json
Comment (tjarrettveracode) [inline Veracode Example.postman_collection.json:196]: this is better

**Review by tjarrettveracode — APPROVED** [review]

**Comment by tjarrettveracode** [comment]
Waiting to merge until pending reviews from @relaxnow and @julz0815 are complete.

File: Veracode Example.postman_collection.json
Comment (relaxnow) [reply Veracode Example.postman_collection.json:3]: If you want to go down the rabbit hold: https://github.com/postmanlabs/postman-app-support/issues/2906
TL;DR it's a meaningless GUID that gets assigned upon import that shouldn't be getting exported with newer versions but it's buggy.

**Comment by relaxnow** [comment]
I tried this but couldn't get it to work, either with my regular account or my service account. I kept getting a 401 Unauthorized:
```
{
    "message": "5f46d643-e134-44d9-bd01-8390650794b1",
    "http_code": 401,
    "http_status": "Unauthorized"
}
```
An application search is working fine. I used the GUID from the application search.

**Comment by relaxnow** [comment]
Also we could use a 'per-request' script for these requests that reminds people to set the 'app_guid':
```
const app_guid = pm.collectionVariables.get('app_guid');
if (!app_guid) {
    throw new Error("Environment does not have an 'app_guid'. Please ensure you have configured an app_guid in the Collection Variables.");
}
```

**Comment by relaxnow** [comment]
@tjarrettveracode did you verify that this works for you?

**Comment by mhowes-veracode** [comment]
> I tried this but couldn't get it to work, either with my regular account or my service account. I kept getting a 401 Unauthorized:
> 
> ```
> {
>     "message": "5f46d643-e134-44d9-bd01-8390650794b1",
>     "http_code": 401,
>     "http_status": "Unauthorized"
> }
> ```
> 
> An application search is working fine. I used the GUID from the application search.

Thanks for testing! I'm also seeing a 401 for both All Static Findings and All SCA Findings. I used these same calls when I wrote REST API Quickstart and they both worked. My API creds are current.

**Comment by mhowes-veracode** [comment]
> 

Great idea! I was planning to add this when I update the REST API quickstart to match these changes. Since we've named the variable `app_guid`, should the description mention that the value comes from the `guid` or `context_guid` properties in the results? See the examples under Steps 2, 7, and 9: https://docs.veracode.com/r/REST_APIs_Quickstart#optional-retrieve-scan-results

**Comment by relaxnow** [comment]
@mhowes-veracode I found and fixed it, the requests were missing the `Authorization` header.
I also added the per-request script.

**Comment by mhowes-veracode** [comment]
> I tried this but couldn't get it to work, either with my regular account or my service account. I kept getting a 401 Unauthorized:
> 
> ```
> {
>     "message": "5f46d643-e134-44d9-bd01-8390650794b1",
>     "http_code": 401,
>     "http_status": "Unauthorized"
> }
> ```
> 
> An application search is working fine. I used the GUID from the application search.

Same for me. I did replace the pre-request script in the collection with the one 

> @mhowes-veracode I found and fixed it, the requests were missing the `Authorization` header. I also added the per-request script.

Brilliant! Yes, both working for me now! Thanks!!

**Review by relaxnow — APPROVED** [review]

**Comment by relaxnow** [comment]
With this it's working now and I added my Approval, but now I'm a contributor so it would probably be best if @julz0815 gave this a whirl.

**Comment by mhowes-veracode** [comment]
Hi @julz0815 , are you able to approve this week? Not urgent, but after we merge I need to update the quickstart for using this collection. Thanks!

**Comment by relaxnow** [comment]
@julz0815 could you take a look at this?

**Pull request merged**
